app/helpers/component_helper.rb

```diff
@@ -0,0 +1,21 @@
+module ComponentHelper
+  def add_option(option, output = nil)
+    return unless option
+    return output if output
+    option
+  end
+
+  def markdown(text)
+    return unless text
+    Redcarpet::Markdown.new(Redcarpet::Render::HTML).render(text).html_safe
+  end
+
+  def markdown_help_url
+    "http://nestacms.com/docs/creating-content/markdown-cheat-sheet"
+  end
```

Comment:
Eventually it would be nice if we could pull this url out into a config var rather that having it hard coded as a string inside of the gem. For this PR I think that it is fine though.